app/views/shared/vote/_new.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= form_for(new_vote, url: votes_path, action: "create") do |f| %>
```

Comment:
do `url` and `action` need to be set here?

Reply:
replace `new_vote` with `parent.votes.build(value: one)` and you can avoid setting the `value`s in the hidden inputs below.